datalad/support/gitrepo.py

```diff
@@ -671,21 +674,174 @@ def git_remote_update(self, name='', verbose=False):
         self._git_custom_command('', 'git remote %s update %s' % (name, v),
                                  expect_stderr=True)
 
-    def git_fetch(self, name, options=''):
+    # TODO: centralize all the c&p code in fetch, pull, push
+    # TODO: document **kwargs passed to gitpython
+    def fetch(self, remote=None, refspec=None, progress=None, all=False, **kwargs):
```

Comment:
Yes. This is a general thing about ironing the Repo classes. Need to figure out gitpython `progress` handleres in general and document it.
Also: These `**kwargs` options for gitpython calls are a cool way to pass custom options to git calls. Meaning: `option=True` translates to `--options`, `b=mybranch` to `-b mybranch`, etc. Exactly what I was aiming for from the beginning (in case you remember).
So, make this consistent and document it for both repo classes is planned for next RF PR. It's nothing about SSH and I won't integrate it this PR. It's the very next thing, if #435 and #482 are merged.